core/src/main/resources/hudson/diagnosis/HudsonHomeDiskUsageMonitor/index.properties

```diff
@@ -23,7 +23,7 @@
 blurb=<tt>JENKINS_HOME</tt> is almost full
 description.1=\
   Your <tt>JENKINS_HOME</tt> ({0}) is almost full. \
-  When this directory completely fills up, it\'ll wreak havoc because Jenkins can\'t store any more data.
+  When this directory completely fills up, it will wreak havoc because Jenkins can\'t store any more data.
```

Comment:
Wouldn't `can't` have the same problem on the same line?

Reply:
Thanks for pointing out this issue.  I've made the edits.

Whenever you are using MessageFormat you should be aware that the single quote character (') fulfils a special purpose inside message patterns. The single quote is used to represent a section within the message pattern that will not be formatted. A single quote itself must be escaped by using two single quotes ('').